src/clojure/com/interrupt/edgar/core/analysis/leading.clj

```diff
@@ -73,6 +90,24 @@
          macd
          ema-signal)))
 
+(defn so
```

Comment:
Can we use the full name here, like `stochastic-oscillator'`.

Reply:
`schochastic-oscillator` is already taken (for now). Besides, I like to keep things short.